install/error-handling.sh

```diff
@@ -1,23 +1,111 @@
 echo "${_group}Setting up error handling ..."
 
+export SENTRY_DSN='https://19555c489ded4769978daae92f2346ca@self-hosted.getsentry.net/3'
+export SENTRY_ORG=self-hosted
+export SENTRY_PROJECT=installer
+
+function send_event {
+  if [[ $DOCKER_PLATFORM == "linux/amd64" ]]; then
+    local sentry_cli="docker run --rm -v \$(pwd):/work -e SENTRY_DSN=$SENTRY_DSN getsentry/sentry-cli"
+  else
+    if ! command -v sentry-cli &> /dev/null; then
+        echo "sentry-cli could not be found, please install it"
+        exit 1
+    fi
+    local sentry_cli=sentry-cli
+  fi
+  command pushd .. > /dev/null
+  $sentry_cli send-event --no-environ -f "$1" -m "$2" --logfile $log_file
+  command popd > /dev/null
+}
```

Comment:
This is confusing to me. Why do we pass `SENTRY_DSN` in the dockerized case but not `SENTRY_ORG` or `SENTRY_PROJECT`?

This also seems like not a great experience on non-linux/amd64, to hard exit during error handling vs. warning early during setup-error-handling and silently passing here. Of course better would be to make it work, how hard would it be to build sentry-cli for ARM? Are we close? Link?

Reply:
I'm also not sure of the advantage of hashing the traceback ourselves vs. sending the traceback as the fingerprint and letting Sentry hash.

Do we want separate status codes to hash together as they do now? It seems a little weird to have variable messages in events that are grouped together.

<img width="1090" alt="variation" src="https://user-images.githubusercontent.com/134455/189398210-af9f6965-c0c6-4981-85ef-162ce28f64ed.png">

Reply:
I think we should have sentry-cli for ARM

https://docs.sentry.io/product/cli/installation/#docker-image

Reply:
a bit confused, is there a reason why we're using the dockerized case for `linux/amd64`?

Reply:
It's unclear to me why we're not passing the rest of the env variables too.

Reply:
actually we only have a pre built image for amd64 😢 

https://hub.docker.com/search?q=sentry-cli

Reply:
I'll answer my own question, dockerized case is preferred or else sentry-cli will need to be installed manually and what you mentioned above chad

Reply:
Yeah, we should pass SENTRY_ORG and SENTRY_PROJECT to docker as well, this was an oversight (added them after writing this I think 😢 )

Agreed testing up front for sentry-cli missing is better. I haven't gotten much information about sentry-cli's build system but that they are working on it, I can ask if there is a link to follow along on the changes.